data/transactions/logic/eval_test.go

```diff
@@ -552,31 +531,14 @@ int 1                   // ret 1
 `, 1)
 }
 
-func TestAddwImpl(t *testing.T) {
-	partitiontest.PartitionTest(t)
-
-	t.Parallel()
-	carry, sum := opAddwImpl(1, 2)
-	require.Equal(t, uint64(0), carry)
-	require.Equal(t, uint64(3), sum)
-
-	carry, sum = opAddwImpl(0xFFFFFFFFFFFFFFFD, 0x45)
-	require.Equal(t, uint64(1), carry)
-	require.Equal(t, uint64(0x42), sum)
```

Comment:
could you please preserve this test as well?

Reply:
Yes, sorry I missed it while changing the format.